### DISCUSSION TITLE
第3节课笔记(13班)

### DISCUSSION THREAD

**Discussion by Lum1104** [InternLM/Tutorial#190] [笔记]
第三节课笔记👇

**Comment by qingyin741** [comment]
https://zhuanlan.zhihu.com/p/676846365

**Comment by BOOMO4** [comment]
https://l055jh6h54r.feishu.cn/wiki/RER0wA8CCiQcBcky1XhcYLeknuf?from=from_copylink

**Comment by zhaoyiz0606** [comment]
https://blog.csdn.net/qq_45699866/article/details/135560110

**Comment by Xrcb** [comment]
https://blog.csdn.net/weixin_64546620/article/details/135560537

**Comment by Taixi-Feng** [comment]
https://zhuanlan.zhihu.com/p/677594136

**Comment by SiliconSisyphus** [comment]
https://www.notion.so/60f62dcdd0a548beacc03a02969d5ac2

**Comment by 5Fbw** [comment]
笔记3
https://fifth-tower-f54.notion.site/ec1e4f21164b4e22b686ff36d38af7a5?pvs=4

**Comment by suwief** [comment]
https://zhuanlan.zhihu.com/p/678185198
